src/net_processing.cpp

```diff
@@ -204,6 +206,15 @@ struct Peer {
     /** Same id as the CNode object for this peer */
     const NodeId m_id{0};
 
+    /** Protects subversion data member */
+    RecursiveMutex m_subver_mutex;
```

Comment:
This has changed from RecursiveMutex to Mutex (since #24079 has been merged a few months ago).
```suggestion
    Mutex m_subver_mutex;
```

Reply:
Thanks. Fixed!